MMM-Afvalwijzer.js

```diff
@@ -92,6 +93,7 @@ Module.register('MMM-Afvalwijzer', {
             case 'PLASTICPLUS':
```

Comment:
'pbd' is the better

Reply:
@gertperdZA 
I have updated that part: https://github.com/gertperdZA/MMM-Afvalwijzer/pull/6/commits/a48f304c32b1e3fd7e259e4c9a85649d58a42572